demos/classification_demo/python/classification_demo.py

```diff
@@ -265,23 +286,27 @@ def main():
             presenter.drawGraphs(frame)
             rendering_start_time = perf_counter()
             frame = draw_labels(frame, classifications, output_transform)
-            if delay or args.no_show:
+            if delay or args.noshow:
                 render_metrics.update(rendering_start_time)
                 metrics.update(start_time, frame)
 
-            if video_writer.isOpened() and (args.output_limit <= 0 or next_frame_id_to_show <= args.output_limit-1):
+            if video_writer.isOpened() and (args.limit <= 0 or next_frame_id_to_show <= args.limit-1):
                 video_writer.write(frame)
 
-            if not args.no_show:
+            if not args.noshow:
                 cv2.imshow('Classification Results', frame)
```

Comment:
```suggestion
                cv2.imshow(__file__, frame)
```

Reply:
@Wovchena Honestly the previous version looks better. I think it should be modified to get only the name of .py file in order to the window has `classification_demo.py` name
![image](https://user-images.githubusercontent.com/74242878/161098628-f88e2370-7f27-4769-851d-e329e7066bbd.png)

Reply:
So, should I revert this commit? @Wovchena

Reply:
Yes, please revert window names in affected demos.